unit-tests/func/func-common.h

```diff
@@ -0,0 +1,93 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "../test.h"
+#include "librealsense2/rs.hpp"
+#include <concurrency.h>
+
+using namespace rs2;
+
+inline rs2::device_list find_devices_by_product_line_or_exit( int product )
```

Comment:
Some of the functions in here are `inline` and some are not

Reply:
done